### PR TITLE
Crossbrowser fix for MC

### PR DESCRIPTION
**Pull request by Vamshi-Muniganti**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by leepavlouhmcts — APPROVED** [review]

**Review by phillip-whitaker — APPROVED** [review]

File: test/e2e/config/crossbrowser.conf.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const config = {
   },
 
 
-  sauceProxy: 'http://proxyout.reform.hmcts.net:8080',  // Proxy for the REST API
+ // sauceProxy: 'http://proxyout.reform.hmcts.net:8080',  // Proxy for the REST API
```
Comment (paulhowes-HMCTS) [inline test/e2e/config/crossbrowser.conf.js:33]: If this isn't needed can it be removed?

File: test/e2e/config/crossbrowser.conf.js [hunk 2]
```diff
@@ -43,36 +43,78 @@ const config = {
...
       sharedTestFiles: false,
       maxInstances: 1
-
     },
 
     {
       browserName: 'MicrosoftEdge',
-      version: '18.17763',
-      platform: 'Windows 10',
+      platform: 'macOS 10.15',
```
Comment (paulhowes-HMCTS) [inline test/e2e/config/crossbrowser.conf.js:88]: Why has this been changed to macOS? Curious...

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
Just a couple of quick queries that might not need changes however I'd like to re-review before approving.

File: test/e2e/config/crossbrowser.conf.js [hunk 3]
```diff
@@ -43,36 +43,78 @@ const config = {
...
+      sharedTestFiles: false,
+      maxInstances: 1
+    },
 
     {
       browserName: 'safari',
-      platform: 'macOS 10.13',
-      version: '11',
+      platform: 'macOS 10.15',
+      version: '13.1',
```
Comment (andywilkinshmcts) [inline test/e2e/config/crossbrowser.conf.js:78]: This is quite an old version of safari, released in 2019. Latest is 14.1

**Review by andywilkinshmcts — APPROVED** [review]
Worth upgrading the safari version

File: test/e2e/config/crossbrowser.conf.js (hunk 1 above)
Comment (Vamshi-Muniganti) [reply test/e2e/config/crossbrowser.conf.js:33]: yes but that might be needed in future if proxy is back if not we remove it in next PRs

File: test/e2e/config/crossbrowser.conf.js (hunk 2 above)
Comment (Vamshi-Muniganti) [reply test/e2e/config/crossbrowser.conf.js:88]: this is all crossbrowser config as per Saucelabs

File: test/e2e/config/crossbrowser.conf.js (hunk 3 above)
Comment (Vamshi-Muniganti) [reply test/e2e/config/crossbrowser.conf.js:78]: thats stable version we are using for tests

**Review by paulhowes-HMCTS — APPROVED** [review]
Approved

**Pull request merged**